README.md

```diff
@@ -7,6 +7,25 @@ The goal of this project is to
 1. train a model to differentiate between the gene expression of healthy individuals and individuals with a disease
 2. apply the model to rare diseases and find how gene expression differs in the disease state
 
+## Method Overview
+The current plan is to use [PLIER](https://github.com/wgmao/PLIER) as a dimensionality reduction method to help the training of a model that
+predicts whether gene expression is healthy or unhealthy. 
+PLIER is well suited for this problem, because it uses prior biological knowlege to constrain the embedding.
+This classifier will then be used to make predictions about diseases, hopefully classifying their gene expression as unhealthy.
```

Comment:
I would mention that you're using the reduced dimensional representation created by PLIER to input into some sort of classifier.  It kinda reads like PLIER is also a classifier because of the "this"